test/src/unit-QueryCondition.cc

```diff
@@ -172,6 +172,151 @@ TEST_CASE("QueryCondition: Test char", "[QueryCondition][char_value]") {
   REQUIRE(query_condition.field_names().count(field_name) == 1);
 }
 
+TEST_CASE(
```

Comment:
Per previous comment -- maybe also have some ints here that are non-zero in all of the byte fields.

Reply:
I will include that in the byte vec value test.